java/vospace.properties.dldb1

```diff
@@ -11,13 +11,13 @@ backend.type.localfs=edu.caltech.vao.vospace.storage.LocalFSStorageManager
 backend.type.swift=edu.caltech.vao.vospace.storage.SwiftStorageManager
 
 # External address of HTTP data server
-server.http.url=http://dldb1.sdm.noao.edu:8080/vospace-2.0/vospace/data
+server.http.url=http://dldb1.datalab.noirlab.edu:8080/vospace-2.0/vospace/data
 
 # External address of transfers endpoint
-server.transfer.url=http://dldb1.sdm.noao.edu:8080/vospace-2.0/vospace/transfers
+server.transfer.url=http://dldb1.datalab.noirlab.edu:8080/vospace-2.0/vospace/transfers
 
 # External address of Datalab Authentication service
-server.auth.url=http://dlsvcs.datalab.noao.edu/auth/
+server.auth.url=http://dlsvcs.datalab.noirlab.edu/auth/
```

Comment:
Preferred URL for production services uses 'https' and should be the proxy e.g. "https://datalab.noirlab.edu/auth" and not the specific 'dlsvcs' instance.